src/core/execute.c

```diff
@@ -434,7 +434,7 @@ int exec_spawn(Unit *unit,
         if (r < 0)
                 return log_unit_error_errno(unit, r, "Failed to set O_CLOEXEC on serialized fds: %m");
 
-        r = log_level_to_string_alloc(log_get_max_level(), &log_level);
+        r = log_level_to_string_alloc(context->log_level_max >= 0 ? context->log_level_max : log_get_max_level(), &log_level);
```

Comment:
Hmm, I'm not sure if this is the right thing to do?

Logs generated by `executor.c`, unlike those by `exec-invoke.c`, are "low-level"/internal logs. They are technically part of pid1 and should not be suppressed by per-unit setting. But, just like @bluca mentioned, I couldn't find anything that uses selinux before `exec_invoke`...

Reply:
I think the forked process _is_ a part of unit process, and the setting LogLevelMax= should control the log level of executor. That's my opinion, but otherwise the meaning of the setting becomes ambiguous.

Reply:
Then, deserializing and setting the log level in `exec_invoke` are unnecessary. Maybe remove that?

Reply:
Indeed. Let's do that later if this is accepted.

Reply:
Given everything, I quite like this solution, it's simple and effective. I tend to consider every log from the fork as part of the unit, because they will be repeated on each fork, and the source is the same process.

Reply:
> Then, deserializing and setting the log level in `exec_invoke` is unnecessary. Maybe remove that?

It is part of the context, so I'd rather just serialize everything rather than pick and choose from those structs, as it's then simpler to reason about and you don't have to keep in mind what is moved across and what isn't.

Reply:
> > Then, deserializing and setting the log level in `exec_invoke` is unnecessary. Maybe remove that?
> 
> It is part of the context, so I'd rather just serialize everything rather than pick and choose from those structs, as it's then simpler to reason about and you don't have to keep in mind what is moved across and what isn't.

OK. But at least it would be good to have a comment there suggesting that it is redundant.